app/controllers/application_controller.rb

```diff
@@ -9,11 +9,11 @@ class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
 
   def is_admin?
```

Comment:
In general this method is confusing because calling is_admin? should not change a variable, it should just return the status. I think Michael suggested doing something like ` logged_in? && current_user.admin?` and then set it afterwards e.g. `@is_admin = is_admin?`  when the functionality calls for it.

Reply:
This would not be a trivial change by any means, as you have to ensure that all use cases of `@is_admin` adheres to this new convention.

Reply:
I think the simple change from the comments would be to eliminate the use of "___ : ___ ?" in favor of simpler logic via "&&"

Reply:
Yeah I tried that, but it broke the Navbar and some test cases as a result. The is_admin? function originally had the @is_admin variable before Iter 1, so we might have to fix logic elsewhere as well. I thought it best to leave it for another story.